src/main/java/com/synopsys/integration/alert/web/config/ConfigActions.java

```diff
@@ -86,6 +86,7 @@ public boolean doesConfigExist(final Long id) throws AlertException {
                 }
             }
 
+            // TODO investigate why this is here instead of in an else if
```

Comment:
The initial setup screen would not display because we had an empty list of configurations but no field models which caused issues generating the UI.  This may no longer be needed.